### PR TITLE
added goveralls

### PR DESCRIPTION
**Pull request by yuce**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,14 +1,18 @@
...
   apt:
     sources:
       - sourceline: 'ppa:masterminds/glide'
     packages:
       - glide
-script: make test
+before_install:
+  - go get github.com/mattn/goveralls
+script:
+  - make vendor && $HOME/gopath/bin/goveralls -service=travis-ci -ignore "internal/internal.go,internal/public.pb.go,internal/private.pb.go"
```
Comment (codysoyland) [inline .travis.yml]: Just to confirm, this doesn't run tests on the whole vendor dir, does it? We had to explicitly exclude it in `make test`.

Comment (yuce) [inline .travis.yml]: As far as I see at: https://coveralls.io/builds/11529694 it doesn't include anything from vendor.

Comment (codysoyland) [inline .travis.yml]: Oh yeah, and the build time on Travis is about the same (roughly 5 minutes). Looks good to me!

Comment (yuce) [inline .travis.yml]: Great!

**Pull request merged**